plugins/inputs/sqlserver/sample.conf

```diff
@@ -11,6 +11,11 @@
     "Server=192.168.1.10;Port=1433;User Id=<user>;Password=<pw>;app name=telegraf;log=1;",
   ]
 
+  ## Timeout for query execution operation
+  ## Note that the timeout for queries is per query not per gather.
+  ## 0 value means no timeout
+  # query_timeout = "5s"
```

Comment:
The value in the example config should show the default value. As-is, this may lead a user to think a 5s timeout is in place by default.

Reply:
done